### DISCUSSION TITLE
How to keep the red color of Input component when isInvalid=true?

### DISCUSSION THREAD

**Discussion by HuseinHQ** [heroui-inc/heroui#2551] [Q&A]
I have a question. In an Input component, there is a slots called classNames. What i want to do is change the color of the inputWrapper to white. This is my code.![image](https://github.com/nextui-org/nextui/assets/94363343/7566d022-3294-450c-bf47-4da782f0a8cf)
The input wrapper color works fine. But the problem is, when my email invalid, the color stay white
My Input:
![Screenshot 2024-03-20 at 13 13 39](https://github.com/nextui-org/nextui/assets/94363343/68bf0403-fb7c-4c41-8893-48d096772aab)
Default Input:
![Screenshot 2024-03-20 at 13 14 58](https://github.com/nextui-org/nextui/assets/94363343/1ad0a3bc-85ca-4074-8822-9378380b06ea)
I want to keep the red color when the email is invalid, how to achieve that?

**Comment by J4v4Scr1pt** [comment]
Please provide code instead of images, easier to test with 😊. Better yet, provide a Stackblitz with the issue.

You could check the attribute:
`inputWrapper: 'bg-white group-data-[invalid=true]:bg-danger-50',`

If this solves your issue please mark it as answered. ✌

**Reply by HuseinHQ** [reply]
it's not working, the inputWrapper background color still white even if the email input is not valid

**Reply by J4v4Scr1pt** [reply]
Strange, works for me, do you have the color "bg-danger-50"? Maybe you have override the configuration?
You should also try with alphaxek suggestion below. 👇

**Reply by HuseinHQ** [reply]
The alphaxek solution changes the entire input color, even if the isInvalid value is false.

**Reply by J4v4Scr1pt** [reply]
Yes, but you could use it like this `!emailValid ? 'danger' : 'primary'`.

Otherwise create a simple Stackblitz with the issue and we can help you better :)

**Comment by alphaxek** [comment]
Hi @HuseinHQ, try passing below props to your `Input` component

```
color="danger"
isInvalid={true}
```

![image](https://github.com/nextui-org/nextui/assets/116849110/7faffcba-bc7e-48b9-b697-ee2e6260c30a)
